package.json

```diff
@@ -10,7 +10,7 @@
     "build:watch": "tsc --watch",
     "clean": "rimraf my-element.{d.ts,d.ts.map,js,js.map} test/my-element.{d.ts,d.ts.map,js,js.map} test/my-element_test.{d.ts,d.ts.map,js,js.map}",
     "lint": "npm run lint:lit-analyzer && npm run lint:eslint",
-    "lint:eslint": "eslint 'src/**/*.ts'",
+    "lint:eslint": "eslint src/**/*.ts",
```

Comment:
```suggestion
    "lint:eslint": "eslint \"src/**/*.ts\"",
```

Reply:
is there any benefit to adding the escaped `"`s, vs. not having any?